ONSdigital/eq-survey-runner
# Ear 1449 hide section navigation summary

### What is the context of this PR?

Jira ticket: https://collaborate2.ons.gov.uk/jira/browse/EAR-449
Remove section navigation from the summary page

### How to review 
- Enable summary page in Author's settings
- Enable section navigation in Author's settings
- To view the changes in Runner, run the command `docker compose build`, followed by `docker compose up`
- View questionnaire in Runner (v2)
- Where section navigation is displayed, it is visible in the right sidebar

**Check:**
- [ ] Section navigation is not displayed on the summary page
- [ ] Section navigation is displayed on all question pages
- [ ] Section navigation is displayed on calculated summary pages
- [ ] Section navigation is displayed on section summary pages
- [ ] Section navigation is displayed on answer summary pages

### Checklist
